.github/workflows/basic-python3.yml

```
@@ -20,7 +20,7 @@ jobs:
           #   * `test_BaseType_Unicode` and `test_nestedStructure` fail due to
           #     DISET's string and unicode types being poorly defined
           - pytest --runslow -k 'not test_BaseType_Unicode and not test_nestedStructure'
-          - pylint -E src/
+          - pylint -E src/ --rcfile=.pylintrc
```

Isn't it the default?
```suggestion
          - pylint -E src/
```

---

If you want a hack for `pytest`, you can add a `git+https://` URL to the `pip` section of the `environment-py3.yaml` and add it to the list of commits to revert for #5193.